modules/administration/pages/actions.adoc

```diff
@@ -40,14 +48,16 @@ You can apply recurring actions on individual Salt clients, or to all clients in
 0 15 2 ? * 7
 ----
 . OPTIONAL: Toggle the [guimenu]``Test mode`` switch on to run the schedule in test mode.
-. Click btn:[Create Schedule] to save, and see the complete list of existing schedules.
+. For actions of type [parameter]``Custom state``, select the states from the list of available states and click btn:[Save Changes]. This will only save the current state selection and not the schedule.
+. In the next pane, drag and drop the selected states to put them in the execution order and click btn:[Confirm].
+. Finally, click btn:[Create Schedule] to save, and see the complete list of existing schedules.
```

Comment:
```suggestion
. Click btn:[Create Schedule] to save, and see the complete list of existing schedules.
```
[Lana used to remove those statement or words.  The user sees that this is the last step of the procedure ;) ]